### DISCUSSION TITLE
SMB share not indexing anymore

### DISCUSSION THREAD

**Discussion by arhills** [music-assistant/support#659] [Q&A] [answered]
MA has been working fine since first install, but I noticed yesterday that it has stopped indexing any files. They are on a smb share and are still connected OK as I can view them through the browse function in MA. I am just guessing there is some DB corruption or similar preventing indexing. Does any one have any clues?

**Comment by OzGav** [comment, marked as answer]
This seems to be an issue so might better of raised there. Also you will probably need to supply more details about why you believe it has stopped indexing. Screen shots, step by step instructions to reproduce the issue for example.

**Reply by arhills** [reply]
As I did not do anything I can't really help with what I changed or did. I may have updated it can't remember. 

I have a NAS share setup as described on this Q & A section. 

Anyhow I managed to fix the issue by re configuring the app. Un ticking the local storage section and ticking spotify then restarting it without a password (so it did not restart) restarting HA then reconfigure as before. It then indexed the files as before.

**Comment by fernplant** [comment]
I've had similar issues with local filesystem not showing up correctly/not syncing and did the same fix. For me, I just put a trailing slash on the media directory in settings, then save, and it syncs again and works

**Comment by arhills** [comment]
After the latest update to 2022.8 it has happened again. The fixes here did not work this time. There is an update showing in hacs but that will not work either. Any ideas for fixes?

**Comment by OzGav** [comment]
Relates to v1
